assets/eip-5299/contracts/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "contracts-starter",
+  "version": "1.0.0",
+  "description": "Template for contract projects.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mudgen/contracts-starter.git"
+  },
+  "author": "Nick Mudge",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/mudgen/contracts-starter/issues"
+  },
+  "homepage": "https://github.com/mudgen/contracts-starter#readme",
+  "devDependencies": {
+    "@nomiclabs/hardhat-ethers": "^2.0.2",
+    "@nomiclabs/hardhat-waffle": "^2.0.1",
+    "chai": "^4.3.4",
+    "dotenv": "^8.6.0",
+    "ethereum-waffle": "^3.4.0",
+    "ethers": "^5.4.1",
+    "hardhat": "^2.4.1",
+    "prettier": "^2.3.2",
+    "prettier-plugin-solidity": "^1.0.0-beta.13",
+    "standard": "^16.0.3"
+  }
+}
```

Comment:
Build tooling isn't appropriate for an EIP.  It would be appropriate for a tutorial, your own GitHub repository, a blog, etc.  Plus, external links not allowed and package.json is basically by definition a bunch of external links.